Sources/SpotHeroAPI/Search/Models/AirportTransportation.swift

```diff
@@ -0,0 +1,45 @@
+// Copyright © 2020 SpotHero, Inc. All rights reserved.
+
+/// Information about transportation between parking facility and the airport.
+public struct AirportTransportation: Codable {
+    private enum CodingKeys: String, CodingKey {
+        case type
+        case contactPhoneNumber = "contact_phone_number"
+        case schedule
+        case images
+        case hoursOfOperation = "hours_of_operation"
+    }
+    
+    /// Defines the type of transportation provided at an airport facility.
+    public enum TransportationType: String, Codable {
+        /// Airport curbside dropoff is available.
+        case curbsideValet = "curbside_valet"
+        
+        /// Free shuttle transportation on demand.
+        case onDemand = "on_demand"
+        
+        /// There is nearby access to public transportation.
+        case publicTransit = "public_transit"
+        
+        /// A free shuttle runs on a scheduled frequency.
+        case scheduled
+        
+        /// Taxi and/or rideshare options are available.
+        case taxi
+    }
+    
+    /// Enum defining the type of airport transportation provided at the facility.
```

Comment:
I feel like the first part is implicit given the strongly typed variable
```suggestion
    /// The type of airport transportation provided at the facility.
```